src/script/campaign/mod/vco-listeners.lua

```diff
@@ -168,6 +168,20 @@ local function vco_def_cop_enable_slaanesh_units()
 	);
 end
 
+local function vco_ksl_ort_disable_flagellant()
```

Comment:
This just unlocks/locks the unit for the faction, but it still has to be reclutable somewhere. Check this PR:
https://github.com/msolefonte/vco/pull/27/files#diff-9b201d85689d8acbea5ba29d459056954ffbf8baeb6bed28e7df8b7aee35e493

And follow the trail from these files:
- https://github.com/msolefonte/vco/blob/main/src/db/building_units_allowed_tables_vco.tsv
- https://github.com/msolefonte/vco/blob/main/src/db/units_to_groupings_military_permissions_tables_vco.tsv

Reply:
https://github.com/msolefonte/vco/pull/131#discussion_r979239048

Do we want to keep this objective? I'll pause on finishing it until we're certain of it.

Reply:
Luminark of Hysh?